static/app/views/dashboardsV2/manage/dashboardList.tsx

```diff
@@ -221,6 +221,7 @@ function DashboardList({
 const DashboardGrid = styled('div')`
   display: grid;
   grid-template-columns: minmax(100px, 1fr);
+  grid-template-rows: repeat(3, max-content);
```

Comment:
I think a max-height on the Dashboard card would be sufficient. It should mirror the same thing in Discover homepage.

Reply:
I'll refactor them after this fix